### PR TITLE
DAO

### PR DESCRIPTION
**Pull request by nearnshaw**

### CODE REVIEW

File: _posts/general/2020-09-16-community-voting.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+## The Lifecycle of a Proposal
+
+A newly created proposal goes through the following stages of revision:
+
+- **Inbox filtering**: The community has 7 days to view it. This phase serves as an initial filter to find proposals that the community has an interest in.
+  _>1 % Minimum Approval >25% Support needed_
+
+- **Security Advisory Board revision**: The Security Advisory Board has 24 hours to review the proposal. This phase serves to protect the DAO from proposals that are badly intended or that could harm the ecosystem. At this stage, the Security Advisory Board can either approve, reject or delay the proposal if they see it fit.
+  _80 % Minimum Approval 100 % Support needed_
```
Comment (nachomazzara) [inline _posts/general/2020-09-16-community-voting.md]: I would remove this line.

Also, I would comment that if after 24 hours the proposal was neither rejected or cancelled. It should be executed and anyone can perform this action.

File: _posts/general/2020-09-16-community-voting.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+- **Community vote**: The community has another 7 days to vote for the proposal. This phase determines if the proposal should be acted upon or not.
```
Comment (nachomazzara) [inline _posts/general/2020-09-16-community-voting.md]: I would start as: `Once someone execute the vote from the Revision step, ....`

File: _posts/general/2020-09-16-community-voting.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  _20 % Minimum Approval & 66 % Support needed_
+
+When the duration of a stage is over and the proposal has enough votes to be approved, then it automatically makes it to the next stage.
+
+Depending on the type of proposal, once approved past the _Community_ stage, the proposal may be automatically implemented by the DAO. Other proposals without on-chain consequences are forwarded to the Decentraland Foundation to be enacted upon.
+
```
Comment (nachomazzara) [inline _posts/general/2020-09-16-community-voting.md:124]: Someone should `enact` the vote after 7 days if it passed. This action could be done by anyone

File: _posts/general/2020-09-16-community-voting.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+### Create a Proposal
+
+You can submit your own proposal to be voted by the community. To do this, click _New Proposal_ and select the type of proposal:
+
+- **Poll**: Ask a question that can be responded with a Yes or a No. This serves as a catch-all alternative for any proposal that doesn’t fit into the other categories.
```
Comment (nachomazzara) [inline _posts/general/2020-09-16-community-voting.md]: This one is hidden right now. We will add it later.

**Review by nachomazzara — COMMENTED** [review]
Amazing job @nearnshaw

**Review by 2fd — APPROVED** [review]

**Pull request merged**